mshorecode/My-Weekend
# Rs styling

## Description
- Overall styling
- updated index.js to remove the option to create a new family if a family has been created
- minimal styling to center "Loading..." message if page is taking long to render
- removed console.warn messages throughout

## Related Issue
#6 #7 

## Motivation and Context
- Quality of life improvements and prevents multiple families for one user

## How Can This Be Tested?
- Sign in to see if it says "Welcome Back, {user}" or "Click button below to create a family"

## Types of changes
